Sources/Contour/Transport/RabbitMQ/Internal/Listener.cs

```diff
@@ -315,6 +316,19 @@ private void Deliver(RabbitDelivery delivery)
                 this.logger.Trace(m => m("Сообщение было обработано в конечных точках: [{0}].", Headers.GetString(delivery.Headers, Headers.Breadcrumbs)));
             }
 
+            var endpoint = this.busContext.Endpoint.Address;
+            var tags = delivery.IsResponse
```

Comment:
It will provoke multiple garbage collections during high load. I think that tags can be created only once.